### PR TITLE
Fix crash in x509_crl when certificate issuer is specified

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Found this while working on #436.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
x509_crl

### CODE REVIEW

File: plugins/modules/x509_crl.py [hunk 1]
```diff
@@ -689,9 +689,7 @@ def _generate_crl(self):
             revoked_cert = revoked_cert.revocation_date(entry['revocation_date'])
             if entry['issuer'] is not None:
                 revoked_cert = revoked_cert.add_extension(
-                    x509.CertificateIssuer([
-                        cryptography_get_name(name, 'issuer') for name in entry['issuer']
```
Comment (felixfontein) [inline plugins/modules/x509_crl.py:693]: `entry['issuer']` is already transformed in line 530.

File: tests/integration/targets/x509_crl/tests/validate.yml [hunk 2]
```diff
@@ -102,3 +102,11 @@
...
           ['commonName', 'Test'],
         ]
+
+- name: Validate CRL 3 info
+  assert:
+    that:
+      - crl_3.revoked_certificates == crl_3_info.revoked_certificates
+      - crl_3.revoked_certificates[0].issuer == [
+          "DNS:ca.example.org",
+        ]
```
Comment (briantist) [inline tests/integration/targets/x509_crl/tests/validate.yml:112]: I was surprised that this plain scalar worked multi-line without `>` or `>-` or something... I did find at least one yaml parser online that didn't like it: https://codebeautify.org/yaml-validator/y22bd59f9

But it appears that validator is wrong; and it clearly works in Ansible regardless, but you may want to change it if only for consistency with the other asserts in the file.

Or not 🤷

**Review by briantist — APPROVED** [review]

File: tests/integration/targets/x509_crl/tests/validate.yml (hunk 2 above)
Comment (felixfontein) [reply tests/integration/targets/x509_crl/tests/validate.yml:112]: According to the YAML definition this should be fine: https://yaml.org/spec/1.2.2/#plain-style
I'd probably rather remove the `>` from the other blocks in this file (but not in this PR) :)

**Comment by felixfontein** [comment]
@briantist thanks for reviewing this!

**Pull request merged**